### DISCUSSION TITLE
"Safari can't establish a secure connection to the server" Magic link

### DISCUSSION THREAD

**Discussion by moizhusnain** [supabase/supabase#962] [Questions] [answered]
Hey everyone! I am struggling with a strange error. I am trying to open a magic link in safari but it is giving me ssl verification error.

![on safari](https://user-images.githubusercontent.com/6430450/112778168-578abb00-905d-11eb-8822-772c6d80198c.png)

I tried it in chrome and the chrome given me "ERR_CONNECTION_RESET". Then i tested it with curl in verbose mode

![with curl](https://user-images.githubusercontent.com/6430450/112778237-8143e200-905d-11eb-9c16-32f68adc9938.png)

and as you can, it is also giving the ssl error. Then to dig further, I tried to open it with openssl

![with openssl](https://user-images.githubusercontent.com/6430450/112778399-e4ce0f80-905d-11eb-9a74-06ac64bd7bdf.png)

So it looks like something is up with the ssl. But you know I mentioned that it is strange, that's because the ssl certificate is valid (https://www.sslshopper.com/ssl-checker.html#hostname=yfzoqbgwhaqcqbueehiu.supabase.co)

![ssl test](https://user-images.githubusercontent.com/6430450/112778516-278fe780-905e-11eb-8ff0-b262d112c3d4.png)

and, I am gonna make it even more interesting, the url is opening outside of my home network. I tried it with https://www.browserling.com/ — it is only happening on my home network, i tried to open the link on my phone

![download](https://user-images.githubusercontent.com/6430450/112778787-b866c300-905e-11eb-8d3e-1cc52046460d.png)

I haven't created an issue because i feel like it is not really an issue with the server end. Just want to get the perspective of the community on it. It is only happening with https://yfzoqbgwhaqcqbueehiu.supabase.co api, I have tried creating another project and it was working fine.

**Comment by nickw** [comment]
I'm experiencing the same exact issue for my instance `qthurzzkbrndhefvfadx.supabase.co`. I can't load the table editor or make any API calls from my client. I get:

`Fetch API cannot load https://qthurzzkbrndhefvfadx.supabase.co/rest/v1/ due to access control checks.
https://qthurzzkbrndhefvfadx.supabase.co/rest/v1/`

and

`Failed to load resource: An SSL error has occurred and a secure connection to the server cannot be made.`

In both Safari and Chrome on macOS Big Sur.

@moizhusnain did you resolve the issue in your app?

**Reply by inian** [reply]
Hi @nickw, I am not able to reproduce this issue on Chrome and Safari Big Sur. Is there a local firewall which might be interfering? 

![image](https://user-images.githubusercontent.com/2155545/123357970-2dce2b00-d59d-11eb-9611-0dd35b76f9c4.png)

**Reply by nickw** [reply]
@inian I don't think so, I'm on my normal home internet that I've been building the app on the past week or so. It was working even earlier today I believe.

**Reply by inian** [reply]
Can you see if https://vercel.com/ loads for you? They use a Lets Encrypt cert as well

**Reply by nickw** [reply]
Yep all good there:

<img width="687" alt="Screen Shot 2021-06-24 at 7 31 53 PM" src="https://user-images.githubusercontent.com/3907/123360872-e2773680-d522-11eb-99f0-957337d466b2.png">

**Reply by nickw** [reply]
Ok it appears my ISP/router had flagged the domain or URL as spyware 🤔 

![IMG_0370](https://user-images.githubusercontent.com/3907/123361186-7517d580-d523-11eb-9035-d94341722530.jpeg)

**Reply by inian** [reply]
😮 Do you know where this info is being retrieved from? We did have an abuse report reported to us before which we took down in minutes, not sure if thats related.

**Reply by nickw** [reply]
My ISP is [xfinity](https://xfinity.com), I'm using their router and their mobile app to manage it. I found this screen buried in the router settings.

**Reply by moizhusnain** [reply]
I ended up creating another project! May be it was a firewall issue, but I tried visiting the address from different internet connections. I used my home broadband, I also tried using my phone network. Nothing worked!

**Comment by inian** [comment, marked as answer]
Tracking the issue [here](https://github.com/supabase/supabase/issues/2096)
